src/whip-client.c

```diff
@@ -321,6 +322,27 @@ static void whip_options(void) {
 	WHIP_LOG(LOG_INFO, "\n");
 }
 
+static gboolean source_events(GstPad* pad,
+														GstObject* parent,
+														GstEvent* event) {
+	gboolean ret;
+
+	switch (GST_EVENT_TYPE(event)) {
+		case GST_EVENT_EOS:
+			ret = gst_pad_event_default (pad, parent, event);
+			// g_print("source_events:GST_EVENT_EOS\n");
```

Comment:
Is the comment a leftover? There's another one below too. Or did you mean them only to be displayed when debugging is higher? In that case you can use a `LOG_VERB` line so that it only appears when debugging 5 is used.

As a side note, there shouldn't be any space after `gst_pad_event_default` either.

Reply:
I just left the logging in there initially... no need for it to stay